text/0000-pack-direct-podman-call.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Pack direct podman call
+- Start Date: 2023-06-16
+- Author(s): dvaumoron
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+A flag `--container-engine` will be added to the `pack` CLI, the allowed flag value will be `docker` (the current functioning) and `podman` to call podman directly (without socket call)
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+This will allow to use pack as a standalone on the OS supported by podman (currently Linux, as Windows and macOS need podman machine), making possible to use pack without installing docker or podman (a tool like skopeo will still be necessary to send image in registries)
+
+# What it is
+[what-it-is]: #what-it-is
+
+Adding `--container-engine podman` to the `pack` command which currently need docker call would allow them to work without docker installed.
```

Comment:
I wonder if `--daemonless` is a better flag name?  Does this imply `--publish` which publishes directly to an OCI registry, rather than using the local daemon for storage.

Reply:
I think a `--daemonless` flag without argument is a better solution, this mode could be used without `--publish` as podman store OCI images in user space (generally in `$HOME/. local/share/containers/storage/`), it depends on the will of the user to run image locally